src/webgpu/web_platform/copyToTexture/ImageBitmap.spec.ts

```diff
@@ -320,43 +371,53 @@ g.test('from_canvas')
       return;
     }
 
-    // The texture format is rgba8unorm, so the bytes per pixel is 4.
-    const bytesPerPixel = 4;
-
-    // Generate original data.
-    const imagePixels = new Uint8ClampedArray(bytesPerPixel * width * height);
-    for (let i = 0; i < width * height * bytesPerPixel; ++i) {
-      imagePixels[i] = i % 4 === 3 ? 255 : i % 256;
-    }
+    // Generate non-transparent pixel data to avoid canvas
+    // different opt behaviour on putImageData()
+    // from browsers.
+    const imagePixels = t.getImagePixels(
+      'rgba8unorm',
+      width,
+      height,
+      'non-transparent', // transparent op
+      'none' // orientation op
+    );
 
     const imageData = new ImageData(imagePixels, width, height);
+
+    // Use putImageData to prevent color space conversion.
     imageCanvasContext.putImageData(imageData, 0, 0);
 
-    const imageBitmap = await createImageBitmap(imageCanvas);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const imageBitmap = await (createImageBitmap as any)(imageCanvas, {
```

Comment:
I don't think the `as any` is needed anymore, now that you have added `as const` on the parameters.

```suggestion
    const imageBitmap = await createImageBitmap(imageCanvas, {
```
Same comment above, at the other usage of `createImageBitmap`.

Reply:
You're right. Remove both `as any`.